app/controllers/api/concerns/errors_handling.rb

```diff
@@ -7,7 +7,6 @@ module ErrorsHandling
         rescue_from StandardError, with: :standard_error
         rescue_from ActionController::ParameterMissing, with: :parameter_missing
         rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
-        rescue_from ActiveRecord::RecordInvalid, with: :record_invalid
```

Comment:
Why is this being removed?

Reply:
It was removed because on the coverage report, it didn't make any hits. When performing manual tests from postman, it was not accessed by an exception either, so since it was not being used, it was better to removed it.

Reply:
Sounds reasonable 👍